### ISSUE TITLE
Graphics level spinner is broken

### ISSUE THREAD

**Issue by Alayan-stk-2** [supertuxkart/stk-code#3977]
See picture below

![BrokenButton2](https://user-images.githubusercontent.com/25536748/60337183-9d0ae780-99a2-11e9-9c65-4872c71aedc9.png)

**Comment by Benau** [comment]
lol Alayan in irc you remote closed connection so we no longer can talk! (Hmm I don't see the same in my linux....)

**Comment by deveee** [comment]
I can't test atm. but maybe try different language or clean config.xml? Anyway I can look at this today.

**Comment by Alayan-stk-2** [comment]
I suspected the first issue about the username on the top right might have been unnoticed, but the graphics level spinner was too big of an issue for that so I searched what I had done wrong after opening this ticket.

I figured out the issue. I pulled master but didn't compile yet. As some differences did appear (the font size spinner as it's an xml file thing), I didn't get immediately that the binary was not correct.

And apparently, the themes have had incompatible changes making things broken when ran with the old code
